src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function f(num) {
```

Comment:
```suggestion
  return function f(num) {
```
next time use better names then `f`